### DISCUSSION TITLE
Android-Termux Pip installing error.

### DISCUSSION THREAD

**Discussion by metin02oktay** [pyinstaller/pyinstaller#6548] [Help]
I got these outputs when I input "pip install pyinstaller"

![resim](https://user-images.githubusercontent.com/23521607/151412570-3aaad773-30e8-4a4b-be96-7c2c1fc5af96.png)

**Comment by rokm** [comment]
> Could not find the program ['strip']

Install `binutils` or whatever package provides `strip` on your system. Without `binutils` and its `objcopy`, you won't get far with PyInstaller on a linux-based system anyway...

**Comment by metin02oktay** [comment]
Can you be more clear please? I'm not high-level like you.

**Reply by bwoodsend** [reply]
`apt-get install binutils`
